packages/console/src/scss/normalized.scss

```diff
@@ -15,3 +17,35 @@ input {
   border: none;
   outline: none;
 }
+
+table {
+  border: 1px solid var(--color-neutral-90);
+  border-radius: _.unit(2);
+  border-spacing: 0;
+  overflow: hidden;
```

Comment:
This might cause problems if we have dropdowns or other floating items in the table, I think.

Reply:
If the overflow is mainly to avoid content overflow rounded corners, we can set the border-radius on thead and tbody

Reply:
make sense

Reply:
`border-radius` doesn't work for `<tr>`